models/zendesk__ticket_metrics.sql

```diff
@@ -55,10 +55,10 @@ select
   ticket_enriched.*,
   ticket_reply_times_calendar.first_reply_time_calendar_minutes,
   ticket_reply_times_calendar.total_reply_time_calendar_minutes,
-  coalesce(ticket_comments.count_agent_comments, 0) as count_agent_comments,
+  coalesce(ticket_comments.count_public_agent_comments, 0) as count_public_agent_comments,
```

Comment:
Same question as above regarding the renaming of the column.

Reply:
With resolution on the previous question is this resolved?

Reply:
Yup!